seongnam/app.js

```diff
@@ -23,12 +25,32 @@ appDataSource.initialize()
     appDataSource.destroy();
     })
 
-const app = express();
 const port = process.env.TYPEORM_PORT;
 app.use(cors());
 app.use(morgan('dev'));
 app.use(express.json());
 
+app.post("/join", async(req,res) =>{
+    try{
+        const data = req.body;
+        console.log(data)
+        const insertQuery = `INSERT INTO users (
+            name,
+            password
+            ) 
+        VALUES (
+            ?, 
+            ?
+        )`;
+        const appData = [ data.name, data.password ];
+        const result = await appDataSource.query(insertQuery, appData);
+        res.status(201).json(result);
+    }catch(error){
+        console.error("Error executing SQL query:", error);
+        res.status(500).json({ message: "Error saving data" });
```

Comment:
상태코드 500 예외처리가 안 되었다는 의미 있을 수 있기 때문에 400번대 코드를 사용하면 좋을 것 같습니다.
```suggestion
        res.status(400).json({ message: "Error saving data" });
```